js/ui/viewSelector.js

```diff
@@ -53,16 +53,18 @@ const ViewsDisplayLayout = new Lang.Class({
     Name: 'ViewsDisplayLayout',
     Extends: Clutter.BinLayout,
 
-    _init: function(entry, allViewActor, searchResultsActor) {
+    _init: function(entry, gc, allViewActor, searchResultsActor) {
         this.parent();
 
         this._entry = entry;
+        this._gc = gc;
```

Comment:
Probably rename this to `discoveryFeed` given that we are not using the old codename anymore.